previews/primer/image_crop_preview.rb

```diff
@@ -7,21 +7,21 @@ class ImageCropPreview < ViewComponent::Preview
     #
```

Comment:
It's probably my fault not updating the migration script properly, but this whole file should move to `previews/primer/alpha` also